ui/src/components/Rules/index.tsx

```diff
@@ -120,17 +120,23 @@ const Rules: React.FC = () => {
             />
           </div>
         </div>
-
-        <div>
+        <h1 className="mb-3 text-lg font-bold text-zinc-200">
+          {'Rules Listing'}
```

Comment:
I think we can simplify the wording to just Rules
```suggestion
          Rules
```

Reply:
Done